mne/channels/layout.py

```diff
@@ -777,9 +780,15 @@ def read_montage(kind, ch_names=None, path=None, scale=True):
     path : str | None
         The path of the folder containing the montage file. Defaults to the
         mne/channels/data/montages folder in your mne-python installation.
-    scale : bool
-        Apply useful scaling for out the box plotting using montage.pos.
-        Defaults to True.
+    unit : 'm' | 'mm'
+        If 'mm', points will be rescaled to 'm'.
+        Default is 'm'.
```

Comment:
I find this a little misleading. I would say something like `Unit of the input file. If not 'm', coordinates will be rescaled to 'm'`. As mentioned already I also think that we should either automatically use the right scaling for builtin montages, or document the fact that this is not done automaticaly.